src/cdk/keycodes/BUILD.bazel

```diff
@@ -1,11 +1,14 @@
 package(default_visibility=["//visibility:public"])
 load("@build_bazel_rules_typescript//:defs.bzl", "ts_library")
+load("@angular//:index.bzl", "ng_module")
 
-# Note: this will need to ng_module if any Angular-specific stuff is added to coercion.
-ts_library(
+
+# TODO(jelbourn): change this back to a ts_library once ng_package supports
```

Comment:
remove the TODO - you need ng_module because you want to produce flat metadata and bundle index which are done by ngc

Reply:
Done